app/celery/scheduled_tasks.py

```diff
@@ -304,3 +306,17 @@ def trigger_link_tests():
     if current_app.config['CBC_PROXY_ENABLED']:
         for cbc_name in current_app.config['ENABLED_CBCS']:
             trigger_link_test.apply_async(kwargs={'provider': cbc_name}, queue=QueueNames.BROADCASTS)
+
+
+@notify_celery.task(name='auto-expire-broadcast-messages')
+def auto_expire_broadcast_messages():
+    expired_broadcasts = BroadcastMessage.query.filter(
+        BroadcastMessage.finishes_at <= datetime.now(),
+        BroadcastMessage.status == BroadcastStatusType.BROADCASTING,
+    )
+
+    for broadcast in expired_broadcasts:
+        broadcast.status = BroadcastStatusType.COMPLETED
+        broadcast.updated_at = datetime.now()
```

Comment:
`updated_at` will get set automatically [when the model is changed](https://github.com/alphagov/notifications-api/blob/03d438b0cc93dec4fe4a9b65b334deee8f48b5d8/app/models.py#L2309), so you could get away without manually setting it here

Reply:
That's good to know. I'm used to setting it manually but I realise that's mostly from doing manual SQL for support purposes. To be fair there is [some inconsistency in where we've applied this](https://github.com/alphagov/notifications-api/blob/03d438b0cc93dec4fe4a9b65b334deee8f48b5d8/app/models.py#L784).

Anyway, I've amended the commit to remove this.